splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -121,6 +121,9 @@ private void configureHandler() throws Exception {
             //Add headers useful for error reporting and metrics
             .setHeader("targetUrl", simple("${headers.metadata[url]}"))
             .setHeader("timeIn", simpleF("%d", System.currentTimeMillis()))
+            
+            //Set Authorization header
+            .setHeader("Authorization", simpleF("Splunk %s", "${headers.metadata[X-Insight-Token]}"))
```

Comment:
@jpramos123 are you sure that this sets the HTTP header? The setHeader sets header on Camel's message exchange.
Here are headers that the Camel HTTP component reads: https://camel.apache.org/components/3.15.x/http-component.html#_message_headers

Have a look at #27, whether that fixes the issue.